recipe/build.sh

```diff
@@ -1,39 +1,88 @@
 #!/bin/bash
 
-osname=`uname`
-if [ $osname == Linux ]; then
-    export CC="gcc"
-    export CXX="g++"
-elif [ $osname == Darwin ]; then
-    export CC="clang"
-    export CXX="clang++"
+set -ex
+
+export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$BUILD_PREFIX/lib/pkgconfig
+export PKG_CONFIG=$BUILD_PREFIX/bin/pkg-config
+
+if [[ $CONDA_BUILD_CROSS_COMPILATION == "1" ]]; then
+  if [[ "${CMAKE_CROSSCOMPILING_EMULATOR:-}" == "" ]]; then
+    # Mostly taken from https://github.com/conda-forge/pocl-feedstock/blob/b88046a851a95ab3c676c0b7815da8224bd66a09/recipe/build.sh#L52
+    rm $PREFIX/bin/llvm-config
+    cp $BUILD_PREFIX/bin/llvm-config $PREFIX/bin/llvm-config
+    export LLVM_CONFIG=${PREFIX}/bin/llvm-config
+  else
+    # https://github.com/mesonbuild/meson/issues/4254
+    export LLVM_CONFIG=${BUILD_PREFIX}/bin/llvm-config
+  fi
+fi
+
+# Always disable GLX, EGL, and GBM to match conda-forge and modern glvnd-based OpenGL usage
+GLX_OPTION="-Dglx=disabled"
+EGL_OPTION="-Degl=disabled"
+GBM_OPTION="-Dgbm=disabled"
+GALLIUM_DRIVERS="-Dgallium-drivers=softpipe,llvmpipe"
+VULKAN_DRIVERS="-Dvulkan-drivers=swrast"
```

Comment:
Is this AI formatted? Could we keep this inline meson setup instead? I find this kind of hard to read.
The comment does not explain why these options are chosen. Best is to remove it. 
A link to https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/meson.options would be useful though.

Reply:
Updeted